styles.css

```diff
@@ -29,58 +45,51 @@ header {
   width: 100vw;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  background-image: url("/images/background.png");
+  align-items: center;
   background-repeat: no-repeat;
-  background-size: 100%;
-  background-position: center;
-  height: auto;
-  padding-left: 24px;
-  padding-right: 24px;
+  background-size: 100% 100%;
+  padding: 0 2rem 0 2rem;
 }
 
 .title {
   font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
+  font-size: 2rem;
   text-align: center;
+  align-self: center;
 }
 
 .name {
-  padding-top: 64px;
   color: #172b4d;
 }
 
 .ocupation {
-  color: #36b37e;
+  animation: color 2s ease infinite alternate;
 }
```

Comment:
Great job @MarcoOquendoC 🤝

- [ ] [Optional] You may please remove this animation. It is not a requirement at the moment and it makes your design different from the Figma template that was provided in the project requirements.